internal/bminventory/inventory_test.go

```diff
@@ -13029,77 +13029,6 @@ var _ = Describe("IPv6 support disabled", func() {
 	})
 })
 
-var _ = Describe("[V2UpdateCluster] IPv6 support disabled", func() {
```

Comment:
removing this, as the other copy contains more tests. see: https://www.diffchecker.com/v50zMwpz